torch_ort_inference/tests/test_openvino_provider_options.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import torch
+import torchvision
+from torch_ort import OpenVINOProviderOptions, ORTInferenceModule
+
+
+class TestOpenVinoProviderOptions:
+    def test_openvino_provider_CPU(self):
+        """Test ORTInferenceModule with CPU backend with FP32 precision"""
+        provider_options = OpenVINOProviderOptions(backend="CPU", precision="FP32")
+        assert isinstance(provider_options, OpenVINOProviderOptions)
+        model = getattr(torchvision.models, "resnet18")(pretrained=True)
+        assert isinstance(model, torch.nn.Module)
+
+        model = ORTInferenceModule(model, provider_options=provider_options)
+        assert isinstance(model, ORTInferenceModule)
+
+        input = torch.rand((1, 3, 256, 256))
+        pred = model(input)
+        assert isinstance(pred, torch.Tensor)
+
+    # Need to add logic to check if iGPU is present and make this skip conditional
```

Comment:
Can remove GPU and VPU checks since it is HW dependent. Generic unit test cases for CPU should be good.

Reply:
So to confirm you'd like me to remove the test cases for GPU and VPU?

Reply:
That's correct.

Reply:
Done, thanks !